src/docker/functions/container/create-container.ts

```diff
@@ -161,3 +163,9 @@ const getUlimits = (ulimits: Ulimits | undefined): DockerodeUlimit[] | undefined
     Soft: value.soft,
   }));
 };
+
+function getVolumes(volumes: Volume[]): { [volume: string]: Record<string, unknown> } | undefined {
```

Comment:
This won't return `undefined` because you're checking if `volumes` is truthy before calling this. 
This function can also be refactored:

```javascript
const getVolumes = (volumes: Volume[]): { [volume: string]: Record<string, never> } => {
  return volumes.reduce((prev, next) => ({ ...prev, [next.name]: {} }), {});
};
```